cmake/onnxruntime_providers_cpu.cmake

```diff
@@ -144,6 +153,12 @@ if (HAS_BITWISE_INSTEAD_OF_LOGICAL)
   target_compile_options(onnxruntime_providers PRIVATE "-Wno-bitwise-instead-of-logical")
 endif()
 
+if(NOT onnxruntime_DISABLE_CONTRIB_OPS)
+  if(USE_NEURAL_SPEED)
+    target_link_libraries(onnxruntime_providers PRIVATE bestla::bestla)
```

Comment:
onnxruntime_add_include_to_target(onnxruntime_providers bestla::bestla)
Also, in the top level CMakeLists add bestla::bestla to onnxruntime_EXTERNAL_LIBRARIES

Reply:
changed.